mage_ai/frontend/components/PipelineDetail/KernelStatus.tsx

```diff
@@ -91,7 +91,7 @@ function KernelStatus({
       onBlur={() => setTimeout(() => setIsEditingPipeline(false), 300)}
       onChange={(e) => {
         setNewPipelineName(e.target.value);
-        pauseEvent(e);
+        e.preventDefault();
```

Comment:
Did you make sure removal of `pauseEvent` doesn't reintroduce some other bug?

Reply:
Will double check before merging

Reply:
> The context menu with only a "Delete" menu seems a bit large. Can we reduce the width a bit?

@johnson-mage added a `compact` prop to `FlyoutMenu` that makes the min width `UNIT * 10`:

<img width="355" alt="image" src="https://user-images.githubusercontent.com/105667442/179328570-e8bb5610-b560-413e-9f56-4830c8d3ccb1.png">

Reply:
everything seems to still work correctly from my testing